docs/source/guide/events.rst

```diff
@@ -212,11 +212,43 @@ Boto3 specific events
 Boto3 emits a set of events that users can register to
 customize clients or resources and modify the behavior of method calls.
 
-Here is the list of events that users of Boto3 can register handlers to:
-
-* ``'creating-client-class'``
-* ``'creating-resource-class'``
-* ``'provide-client-params'``
+Below is a table of events that users of Boto3 can register handlers to. More information
+about each event can be found in the corresponding sections below:
+
++-----------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+
+| Event Name                  | Order | Emit Location                                                                                                                        |
++=============================+=======+======================================================================================================================================+
+| ``creating-client-class``   | 1     | `Location <https://github.com/boto/botocore/blob/aa63a377c6c9a78c87f55e5c0fc6ca6643198f3b/botocore/client.py#LL194C1-L198C10>`_      |
```

Comment:
I agree. I'll be replacing these links with ones of the following structure:

`https://github.com/search?q=repo%3A<github_repo>+<search_text>+path%3A<file_path_regex>&type=code`

Where:
> Note: All values are URL encoded.
* `<github_repo>` is the GitHub repo name such as `boto/botocore` or `boto/boto3`.
* `<search_text>` is the text/code we're searching for. This will probably be the event name. We may add more surrounding code if we need more context to narrow the search.
* `<file_path_regex>` is a regex for the path of the file we want to search for. Example: `/^botocore\/client\.py/`.

Example:
https://github.com/search?q=repo%3Aboto%2Fbotocore+creating-client-class+path%3A%2F%5Ebotocore%5C%2Fclient%5C.py%2F&type=code

Reply:
Amazing thank you Jonathan!